src/components/datetime/QDatetime.js

```diff
@@ -26,25 +26,35 @@ export default {
   mixins: [FrameMixin],
   props: extend(
     input,
-    inline
+    inline,
+    {
+      popover: Boolean,
+      modal: Boolean
```

Comment:
do you think it would be better to have a "mode" prop instead of the two? it also offers room for expansion later & it will be less code

Reply:
Possibly, I had originally combined the two, but for now, it seems seperating them would be simpler and more clear for user declaration
`<q-datetime  popover ...`
`<q-datetime  modal ...`
instead of
`<q-datetime  mode="popover" ...`

But I leave the decision to you.

Reply:
let's go with the two props. one thing I think it's gonna bite is that devs will want same behavior for QColor too. can we make this into a mixin and use it in both datetime and color? let me know if you'd rather do this only for datetime.

Reply:
I have alot more work on QDateTime before I move onto another.  But yeah, when other components are changed to combine these concepts, we can make a common mixin.  I will simply follows your design decisions are that time. :)

Reply:
But if you feel it's a priority now to harmonize QColor with this QDateTime feature , tell me and I can do it! :)

Reply:
it would be a priority in order to maintain consistency. but it's your choice. however it will fit your schedule. I can do it tomorrow morning so you can focus on more datetime goodies if you want.